include/aktualizr-lite/tuf/aklite-tuf.h

```diff
@@ -0,0 +1,248 @@
+#ifndef AKLITE_TUF_API_H_
+#define AKLITE_TUF_API_H_
+
+#include <iostream>
+#include <string>
+
+#include <boost/algorithm/string.hpp>
+#include <boost/filesystem.hpp>
+#include <boost/program_options.hpp>
+#include <boost/property_tree/ptree.hpp>
+
+#include "crypto/keymanager.h"
+#include "http/httpclient.h"
+#include "json/json.h"
+#include "libaktualizr/config.h"
+#include "storage/invstorage.h"
+#include "uptane/fetcher.h"
+#include "uptane/imagerepository.h"
+
+#include "aktualizr-lite/tuf/aklite-tuf.h"
+
+namespace aklite {
+namespace tuf {
+
+// DISCUSS: multiple fetch methods, or a single fetchRole?
+// DISCUSS: return string vs return Json
```

Comment:
I incline towards returning json since I believe that all clients would parse a string to json anyway.